Assets/Global/Scripts/Player/Attacks/TailAttack.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+
+public class TailAttack : Attack
+{
+    [HideInInspector]
+    public Player player;
+
+    public void Awake()
```

Comment:
Unsure if its best practice to use the GlobalReference in the Awake() method, as i think the references in GlobalReference themselves are being set on awake. So there might be a (ever so small) chance the references end up not existing yet. But @fabiothomas probably knows this better and can tell me if im wrong 👍